docs/tutorials/stateful-application/cassandra.md

```diff
@@ -1,374 +1,134 @@
-computer.
-
-## Cassandra Docker
-
-The pods use the [```gcr.io/google-samples/cassandra:v12```](https://github.com/kubernetes/examples/blob/master/cassandra/image/Dockerfile)
-image from Google's [container registry](https://cloud.google.com/container-registry/docs/).
-The docker is based on `debian:jessie` and includes OpenJDK 8. This image
-includes a standard Cassandra installation from the Apache Debian repo.  Through the use of environment variables you are able to change values that are inserted into the `cassandra.yaml`.
-
-| ENV VAR       | DEFAULT VALUE  |
-| ------------- |:-------------: |
-| CASSANDRA_CLUSTER_NAME | 'Test Cluster'  |
-| CASSANDRA_NUM_TOKENS  | 32               |
-| CASSANDRA_RPC_ADDRESS | 0.0.0.0          |
-
-## Quickstart
-
-If you want to jump straight to the commands we will run,
-here are the steps:
-
-```sh
-#
-# StatefulSet
-#
+{% capture overview %}
+This tutorial will show you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
+{% endcapture %}
 
-# clone the example repository
-git clone https://github.com/kubernetes/examples
-cd examples
+{% capture objectives %}
+* Create and Validate a Cassandra headless service.
+* Use a StatefulSet to create a Cassandra ring.
+* Validate the StatefulSet.
+* Modify the StatefulSet.
+* Delete the StatefulSet and its pods.
+{% endcapture %}
 
-# create a service to track all cassandra statefulset nodes
-kubectl create -f cassandra/cassandra-service.yaml
+{% capture prerequisites %}
+You should already have a basic familiarity with [Pods](#), [Services](#), and [StatefulSets](#), but to complete this tutorial you only need: 
 
-# create a statefulset
-kubectl create -f cassandra/cassandra-statefulset.yaml
+* To [Install and Configure](/docs/tasks/tools/install-kubectl/) the `kubectl` command line
 
-# validate the Cassandra cluster. Substitute the name of one of your pods.
-kubectl exec -ti cassandra-0 -- nodetool status
+* To have a v1.5 or later Kubernetes Cluster running
 
-# cleanup
-grace=$(kubectl get po cassandra-0 -o=jsonpath='{.spec.terminationGracePeriodSeconds}') \
-  && kubectl delete statefulset,po -l app=cassandra \
-  && echo "Sleeping $grace" \
-  && sleep $grace \
-  && kubectl delete pvc -l app=cassandra
+Check out the the [getting started guides](/docs/setup/pick-right-solution/) if you don’t already have a cluster. 
 
-#
-# Resource Controller Example
-#
-
-# create a replication controller to replicate cassandra nodes
-kubectl create -f cassandra/cassandra-controller.yaml
-
-# validate the Cassandra cluster. Substitute the name of one of your pods.
-kubectl exec -ti cassandra-xxxxx -- nodetool status
+Warning: [Minikube](/docs/getting-started-guides/minikube/) defaults to 1024MB of memory and 1 CPU which results in an insufficient resource errors. 
```

Comment:
Not your fault, but we need better callouts for warnings like these.  Filed the following issue: https://github.com/kubernetes/kubernetes.github.io/issues/4458

At the very least, please bold "Warning"